### ISSUE TITLE
Normalize test set using training set

### ISSUE THREAD

**Issue by walbermr** [wmr1/frame-forecast#5]
hint by: marcelsan
